.github/workflows/change-file-build-new-actions-vm.yml

```diff
@@ -0,0 +1,52 @@
+name: Change File Build New Actions VM
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+        - "ci/gha-runner-vm/**"
+
+jobs:
+  build:
+    name: Build new Actions VM
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.24.x'
+      - name: Install dependencies
+        run: cd ci/gha-runner-vm && go get .
+      - name: Install Packer
+        run: |
+          curl -LO https://releases.hashicorp.com/packer/1.12.0/packer_1.12.0_linux_amd64.zip
+          unzip packer_1.12.0_linux_amd64.zip
+          sudo mv packer /usr/local/bin/
+          rm packer_1.12.0_linux_amd64.zip
+          packer plugin install github.com/hashicorp/oracle
+      - name: Install OCI CLI
+        env:
+          OCI_CLI_USER: ${{ secrets.OCI_CLI_USER }}
+          OCI_CLI_TENANCY: ${{ secrets.OCI_CLI_TENANCY }}
+          OCI_CLI_FINGERPRINT: ${{ secrets.OCI_CLI_FINGERPRINT }}
+          OCI_CLI_KEY_CONTENT: ${{ secrets.OCI_CLI_KEY_CONTENT }}
+          OCI_CLI_REGION: ${{ secrets.OCI_CLI_REGION }}
+          OCI_INSTANCE_ID: ${{ secrets.OCI_INSTANCE_ID }}
+        run: |
```

Comment:
I'm not against this, but we could also use oracle actions: https://github.com/oracle-actions/run-oci-cli-command